### DISCUSSION TITLE
whether Prisma supports dynamically creating schema or not?

### DISCUSSION THREAD

**Discussion by Malenconiaprincep** [prisma/prisma#7545] [Q&A] [answered]
hi， Create a schema like KNEXJS, I want to dynamic create that is so efficient

**Comment by ryands17** [comment, marked as answer]
@Malenconiaprincep 👋 

Currently Prisma doesn't support this but it would be great if you could add a 👍 on this request so that we can look into this:
https://github.com/prisma/prisma/issues/227

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
